src/humidity.toit

```diff
@@ -13,6 +13,8 @@ interface HumidityService:
       --minor=0
 
   read -> float
+  close -> none
+
   static READ-INDEX ::= 0
```

Comment:
```suggestion
  read -> float
  static READ-INDEX ::= 0

  close -> none
```

Reply:
The statics are going to be moved out of the class in a follow-up PR. Leaving in place to avoid merge conflicts.